stablehlo/tests/vhlo/stablehlo_legalize_to_vhlo.1_10_0.mlir

```diff
@@ -1,10 +1,10 @@
-// RUN: stablehlo-opt --mlir-print-op-generic %s.bc | FileCheck %s
-// RUN: stablehlo-translate --deserialize %s.bc | stablehlo-translate --serialize --target=1.10.0 | stablehlo-opt --mlir-print-op-generic | FileCheck %s
-// RUN: stablehlo-translate --deserialize %s.bc | stablehlo-opt > %t.0
-// RUN: stablehlo-opt --strip-debuginfo %s > %t.1
+// RUN: stablehlo-opt --stablehlo-legalize-to-vhlo --mlir-print-op-generic --split-input-file %s | FileCheck %s
+// RUN: stablehlo-translate --serialize --target=current %s | stablehlo-translate --deserialize | stablehlo-opt > %t.0
+// RUN: stablehlo-opt %s > %t.1
```

Comment:
Un-revert this file. This was a bugfix, LHS here is correct

Reply:
done.